docs/threat-model.md

```diff
@@ -8,7 +8,13 @@ netperf is a framework and set of tools for measuring network (and related compo
 
 #### netperf Repository
 
-The netperf repository contains the scripts and yaml for the netperf framework and tools. It also contains various secrets to access an Azure Subscription.
+The netperf repository contains the scripts and yaml for the netperf framework and tools. It also contains the following secrets, stored in GitHub Respository secrets:
+
+- `AZURE_CLIENT_ID` - Used for accessing the Azure Subscription
+- `AZURE_SUBSCRIPTION_ID` - Used for accessing the Azure Subscription
+- `AZURE_TENANT_ID` - Used for accessing the Azure Subscription
+- `PERSONAL_ACCESS_TOKEN` - Used to access the GitHub CLI
```

Comment:
Potential repair item:
Move all secrets to Azure key vault and make access conditional on federated token access.

Reply:
Example:
https://github.com/microsoft/ebpf-for-windows/blob/c4b7fe423402fa66ff243ec02a451c354ca4be83/.github/workflows/upload-perf-results.yml#L80C1-L87C1

Reply:
Opened https://github.com/microsoft/netperf/issues/162. Thanks!